src/coreclr/vm/jitinterface.cpp

```diff
@@ -10970,6 +10970,12 @@ void CEEJitInfo::WriteCode(EEJitManager * jitMgr)
     UnwindInfoTable::PublishUnwindInfoForMethod(m_moduleBase, m_CodeHeader->GetUnwindInfo(0), m_totalUnwindInfos);
 #endif // defined(TARGET_AMD64)
 
+    {
+        ExecutableWriterHolder<BYTE *> gcInfoWriterHolder(m_CodeHeader->GetGCInfoAddr(), sizeof(void *));
```

Comment:
`ExecutableWriterHolder` is not a cheap operation. `WriteCodeBytes` has one already. Can we refactor such that we have just one ExecutableWriterHolder for both operations?